.github/workflows/vault-sync-app-body.yml

```diff
@@ -0,0 +1,56 @@
+name: runner-config
+on:
```

Comment:
Should be able dispatch manually as well

Reply:
So add workflow_dispatch?

Reply:
yes

Reply:
Will the workflow_dispatch need inputs as well?